neqo-bin/src/client/http3.rs

```diff
@@ -222,7 +222,9 @@ impl<'a> super::Handler for Handler<'a> {
                     }
                 }
                 Http3ClientEvent::StateChange(Http3State::Connected)
-                | Http3ClientEvent::RequestsCreatable => {
+                | Http3ClientEvent::RequestsCreatable
+                | Http3ClientEvent::ZeroRttRejected => {
+                    qinfo!("{event:?}");
```

Comment:
:+1: good idea!

Reply:
I think for the interop runner, only h09 matters.

We should also check that Fx is doing the right thing.

Reply:
In Fx, we rewind the stream to the state it was in before attempting 0RTT. The code that handles the ZeroRttRejected event is [here](https://searchfox.org/mozilla-central/rev/527d691a542ccc0f333e36689bd665cb000360b2/netwerk/protocol/http/Http3Session.cpp#608). Essentially, we reset the [offset of the request stream](https://searchfox.org/mozilla-central/rev/527d691a542ccc0f333e36689bd665cb000360b2/netwerk/protocol/http/nsHttpTransaction.cpp#3079) and [all internal states of HTTP3Stream](https://searchfox.org/mozilla-central/rev/527d691a542ccc0f333e36689bd665cb000360b2/netwerk/protocol/http/Http3Stream.cpp#465-479).

I think this behavior is correct.

Reply:
> I think for the interop runner, only h09 matters.

Same on h09, here I would expect the client to eventually call `self.download_urls(client)` given the match on:

``` rust
 ConnectionEvent::StateChange(
    State::WaitInitial | State::Handshaking | State::Connected,
 ) => {
```

Reply:
I think the issue (for both h3 and h09) is that already-tried URLs are popped off the list and are not retried. I have a fix for this now in the PR; CI is running.